.github/workflows/release.yml

```
@@ -0,0 +1,213 @@
+            fi
+          done
+          
+          # Rename fat APK
+          if [ -f "app-release.apk" ]; then
+            mv "app-release.apk" "dyip-v${VERSION}-universal.apk"
+          fi
+          
+          ls -lh
+      
+      - name: Upload APKs to Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.8)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)